tests/parametric/test_sampling_span_tags.py

```diff
@@ -0,0 +1,118 @@
+import json
+
+import pytest
+from utils import bug, scenarios
+from utils.parametric.spec.trace import MANUAL_DROP_KEY  # noqa
+from utils.parametric.spec.trace import MANUAL_KEEP_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_AGENT_PRIORITY_RATE  # noqa
+from utils.parametric.spec.trace import SAMPLING_DECISION_MAKER_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_PRIORITY_KEY  # noqa
+from utils.parametric.spec.trace import SAMPLING_RULE_PRIORITY_RATE  # noqa
+from utils.parametric.spec.trace import Span  # noqa
+from utils.parametric.spec.trace import find_span_in_traces  # noqa
+
+
+def _get_parent_and_child_span(test_agent, test_library):
+    with test_library:
+        with test_library.start_span(name="parent", service="webserver") as parent_span:
```

Comment:
nit: parent_span shadows the span returned on line 25. Using a different name here could improve maintainability 

```suggestion
        with test_library.start_span(name="parent", service="webserver") as ps:
```